session4/src/com/itschool/homework/Challenge3.java

```diff
@@ -1,33 +1,40 @@
 package com.itschool.homework;
 
+import java.util.Scanner;
 public class Challenge3 {
 
     public static void main(String[] args) {
 
-        boolean a = true;
-        boolean b = false;
+//        boolean a = true;
+//        boolean b = false;
         //logical complement operators
 
-        System.out.println(!a);
-        System.out.println(!b);
-
-        int c = 1;
-        int d = 2;
-
-        System.out.println(c>d);
-
-        int e = -1;
-        int f = +3;
-
-        System.out.println(e>0);
-        System.out.println(f>0);
-
-        boolean w = true;
-        boolean z = false;
+//        System.out.println(!a);
+//        System.out.println(!b);
+//
+//        int c = 1;
+//        int d = 2;
+//
+//        System.out.println(c>d);
+
+//        int e = -1;
+//        int f = +3;
+//
+//        System.out.println(e>0);
+//        System.out.println(f>0);
+
+//        boolean w = true;
+//        boolean z = false;
         //negation operators
-
-        System.out.println(!w);
-        System.out.println(!z);
+//
+//        System.out.println(!w);
+//        System.out.println(!z);
+
+        Scanner input = new Scanner(System.in);
+        System.out.println("Please insert the first boolean: ");
+        boolean a = input.nextBoolean();
+        System.out.println("Please insert the second boolean: ");
+        boolean b = input.nextBoolean();
```

Comment:
that's cool, but you don't do anything with `a` and `b`. Make some operations with them and display their result.